go.mod

```diff
@@ -6,20 +6,19 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v1.2.3
 	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/google/uuid v1.2.0
+	github.com/google/uuid v1.3.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.20.1
-	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
-	github.com/openshift/client-go v0.0.0-20220915152853-9dfefb19db2e
-	github.com/openshift/library-go v0.0.0-20220920133651-093893cf326b
+	github.com/openshift/api v0.0.0-20220628012702-9368b92de0ab
```

Comment:
Why is this going back to a version from june? When it is up to date at present?